frontend/package.json

```
@@ -103,6 +103,9 @@
   "overrides": {
     "marked-gfm-heading-id": {
       "marked": "$marked"
+    },
+    "@analogjs/platform": {
+      "shiki": "^2.1.0"
```

```suggestion
      "shiki": "$shiki"
```

This automatically inherits the value from the dependencies list and always keeps it consistent.